README.md

```diff
@@ -47,19 +47,19 @@ Alfa will run in any [ECMAScript 2015](https://www.ecma-international.org/ecma-2
 
 ## Building
 
-In order to build Alfa, a recent version (>= 9) of [Node.js](https://nodejs.org/) is required in addition to the [Yarn](https://yarnpkg.com/) package manager. For builds, [npm](https://www.npmjs.com/) is **not** supported as Alfa makes use of [Yarn workspaces](https://yarnpkg.com/lang/en/docs/workspaces/). Once Node.js and Yarn have been installed, go ahead and install the Alfa development dependencies:
+In order to build Alfa, a recent version (>= 9) of [Node.js](https://nodejs.org/) is required in addition to the [Yarn](https://yarnpkg.com/) package manager. For builds, [npm](https://www.npmjs.com/) is **not** supported as Alfa makes use of [Yarn workspaces](https://yarnpkg.com/lang/en/docs/workspaces/). Once Node.js and Yarn have is installed, go ahead and install the Alfa development dependencies:
```

Comment:
Not "is", but "are", if anything. The previous version was fine though.

Reply:
Good catch. I will change it to `are`.